test/namenode/CreationTest.cc

```diff
@@ -0,0 +1,128 @@
+// Copyright 2017 Rice University, COMP 413 2017
+
+#include "NameNodeTest.h"
+
+std::vector<std::string> createTestString(
+        int num_files, int num_dirs
+) {
+    std::vector<std::string> generated_files;
+    std::string prefix = std::to_string(num_files) + "_" + std::to_string(num_dirs) + "_";
+    // Create the intermediate directories
+    for (int i = 0; i < num_files; i++) {
+        std::string temp = "";
+        for (int d = 0; d < num_dirs; d++) {
+            temp = temp + "/" + prefix + "f" + std::to_string(i) + "_d" + std::to_string(d);
+        }
+        // Now add the actual file
+        temp = temp + "/" + prefix + "file";
+        generated_files.push_back(temp);
+    }
+    return generated_files;
+}
+
+TEST_F(NamenodeTest, createBasicFile) {
```

Comment:
if you're not gonna use the NamenodeTest fixture anymore, all the tests in this target can be `TEST` instead of `TEST_F`.

Reply:
We are using the fixture, it sets up all of the zk client stuff.

Reply:
nice i just missed it in the .h file